core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -583,48 +587,4 @@ private static PathConfiguration loadPathConfiguration(GetConfigurationPResponse
     LOG.info("Alluxio client has loaded path level configurations");
     return PathConfiguration.create(pathConfs);
   }
-
-  private static boolean shouldLoadClusterConfiguration(AlluxioConfiguration conf) {
-    return conf.getBoolean(PropertyKey.USER_CONF_CLUSTER_DEFAULT_ENABLED)
```

Comment:
Both clients and Alluxio servers other than master will load configurations from master.
For clients, due to configuration propagation introduced in this PR, it will refresh updated configurations in master.
For other servers, currently, it will only load configurations from master once.

Reply:
what do you think about introducing the load type instead of multiple properties for enable cluster defaults and another property for whether to refresh?

Reply:
sg